ktor-client/ktor-client-core/common/src/io/ktor/client/plugins/sse/builders.kt

```diff
@@ -0,0 +1,177 @@
+/*
+* Copyright 2014-2023 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+*/
+
+package io.ktor.client.plugins.sse
+
+import io.ktor.client.*
+import io.ktor.client.plugins.*
+import io.ktor.client.request.*
+import io.ktor.http.*
+import io.ktor.sse.*
+import kotlinx.coroutines.*
+
+/**
+ * Installs the [SSE] plugin using the [config] as configuration.
+ */
+public fun HttpClientConfig<*>.SSE(config: SSEConfig.() -> Unit) {
+    install(SSE) {
+        config()
+    }
+}
+
+/**
+ * Opens a [ClientSSESession].
+ */
+public suspend fun HttpClient.serverSentEventsSession(
```

Comment:
should we also rename it in api to `sseSession`?

Reply:
There is also a similar function with the name `sseSession `. I thought to leave both variants like in WebSocket plugin, where it is `ws` and `websocket`